### PR TITLE
fix contentEditable warning

### PR DESCRIPTION
**Pull request by linkkingjay**
See https://github.com/facebook/draft-js/issues/53. The issue has been fixed in the React repo: https://github.com/facebook/react/pull/6112 .

### CODE REVIEW

File: src/component/base/DraftEditor.react.js [hunk 1]
```diff
@@ -262,6 +262,7 @@ class DraftEditor
             ref="editor"
             role={readOnly ? null : (this.props.role || 'textbox')}
             spellCheck={allowSpellCheck && this.props.spellCheck}
+            suppressContentEditableWarning={!readOnly}
```
Comment (hellendag) [inline src/component/base/DraftEditor.react.js]: Couldn't this always just be `true`?

Comment (linkkingjay) [inline src/component/base/DraftEditor.react.js]: @hellendag Of course it can. Fixed.

**Comment by adcoolguy** [comment]
We have the same warning even after using this new attribute. Are we missing something?

                                <div className="control-group">
                                    <label htmlFor="desc" className="col-sm-2 control-label">Description:</label>
                                    <div className="col-sm-10">
                                        <Editor ref="desc" suppressContentEditableWarning editorState={editorState} onChange={this.onChange} />
                                    </div>
                                </div>

We use React 14.0. Thanks!

**Comment by tasti** [comment]
@adcoolguy: The `suppressContentEditableWarning` prop will be added into React 15.0 (as mentioned https://github.com/facebook/react/issues/5837).

Also, you won't have to place the prop on your `Editor` component; this PR is placing it internally for you.

**Comment by spicyj** [comment]
I actually ended up getting this as part of #179. Thanks for sending this in though!

**Pull request closed without merging**